### ISSUE TITLE
'api.search_submission_comment_ids' function only returns one comment id for a post with multiple comments

### ISSUE THREAD

**Issue by jingjieli95** [mattpodolak/pmaw#9]
Hi!

I have been trying pmaw to request pushshift reddit data. I am trying to get all the comment ids given a post/submission id using api.search_submission_comment_ids. An example of my code is: 

```
list_id = ['6uey5x'] 
comment = api.search_submission_comment_ids(ids=list_id)
```
However, it only returns one comment id from this requested submission id: `['dls663o']`. The request should have returned multiple comment ids according to the pushshift api: https://api.pushshift.io/reddit/submission/comment_ids/6uey5x

I also tried configuring the available options like memsafe, which didn't help. Interestingly, if I try to repeat the post id in the input, such as:
```
list_id = ['6uey5x', '6uey5x', '6uey5x', '6uey5x'] 
comment = api.search_submission_comment_ids(ids=list_id)
```
It gives more comment ids from the corresponding post: `['dls663o', 'dls67q6', 'dls6fgv', 'dls6fqj']`. 

Could you help look into it? I feel like this workaround might have not fully utilized the pipeline.

I really apprecaite your effort and this great tool!

**Comment by mattpodolak** [comment]
Hi @jingjieli95 thanks for opening this issue, I'll look into this

**Comment by mattpodolak** [comment]
I found the issue, this is due to a limit miscalculation for `ids`. I'll push out a fix in an hour or so

**Comment by mattpodolak** [comment]
Closed this issue as this has been fixed in `v1.0.4`, please open another issue if you run into any problems
